src/lib/eas/fetchAttestation.tsx

```diff
@@ -0,0 +1,34 @@
+import { getProjects } from "@/lib/eas/getProjects";
+import getDecodedAttestationData from "@/lib/eas/getDecodedAttestationData";
+import MockData from "@/components/Project/project.json";
+import { getAddress } from "@/lib/eas/getAddress";
+
+export const fetchAttestation = async (address: any, id: string) => {
+  let refUid = "";
+  let uid = id;
+  let accountAddress = address;
+
+  const queryParam = address ? `?address=${address}` : "";
+  const projects: any = await getProjects(queryParam);
```

Comment:
if this function is to retrieve one, single, attestation, why are we getting ALL projects for the user?
- we already know the `refUid` right?